ext/spl/spl_directory.stub.php

```diff
@@ -190,101 +190,101 @@ class SplFileObject extends SplFileInfo implements RecursiveIterator, SeekableIt
     /** @param resource|null $context */
     public function __construct(string $filename, string $mode = "r", bool $useIncludePath = false, $context = null) {}
 
-    /** @return void */
-    public function rewind() {}
+    /** @tentative-return-type */
+    public function rewind(): void {}
 
-    /** @return bool */
-    public function eof() {}
+    /** @tentative-return-type */
+    public function eof(): bool {}
 
-    /** @return bool */
-    public function valid() {}
+    /** @tentative-return-type */
+    public function valid(): bool {}
 
-    /** @return string */
-    public function fgets() {}
+    /** @tentative-return-type */
+    public function fgets(): string {}
 
-    /** @return string|false */
-    public function fread(int $length) {}
+    /** @tentative-return-type */
+    public function fread(int $length): string|false {}
 
-    /** @return array|false */
-    public function fgetcsv(string $separator = ",", string $enclosure = "\"", string $escape = "\\") {}
+    /** @tentative-return-type */
+    public function fgetcsv(string $separator = ",", string $enclosure = "\"", string $escape = "\\"): array|false|null {}
```

Comment:
Ugh. So apparently `null` is returned on EOF and `false` if parsing the line fails. Documentation only mentions false. Wonder if we should change the implementation?

Reply:
I fixed this in https://github.com/php/php-src/commit/188b1d4c7c7b3482584e248522d94e06ba616a1c.

Reply:
Did you forget to update the stub itself? I see that only the arginfo hash changed in your commit.

Reply:
No problem though, I'll change this line in any case with my next commit.